terraform_deploy.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+terraform init
+terraform apply -var="APP-PREFIX=$PRODUCTION_ENV_PREFIX" -var="CLIENTID=$CLIENT_ID" -var="CLIENTSECRET=$CLIENTSECRET"  -auto-approve
+curl -dH -X POST "$(terraform output -raw cd_webhook)"
```

Comment:
Similarly we want the build to fail if this doesn't work, by default curl considers any response a success, including 4xx and 5xx.
```suggestion
curl -dH -X POST --fail "$(terraform output -raw cd_webhook)"
```